spt/OrangeBox/scripts/tests/test_item.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <string>
+#include <vector>
+
+namespace scripts
+{
+
+	struct TestItem
+	{
+		TestItem(int tick, int no, const std::string& data);
+
+		int Tick;
+		int TrackerNo;
```

Comment:
I think it's better to store the tracker name rather than its number (which depends on internal insertion order detail).